lib/session/call-session.js

```diff
@@ -969,6 +970,91 @@ class CallSession extends Emitter {
     }
   }
 
+  /**
+   * perform live call control - create rest:dial
+   * @param {obj} opts create call options
+   */
+  async _lccCallDial(opts) {
+    try {
+      const restDialUrl = `${this.srf.locals.serviceUrl}/v1/createCall`;
+      await this._validateCreateCall(opts);
+      const resp = bent('POST', 201)(restDialUrl, opts);
+      this.logger.info(resp.body, 'successfully create outbound call');
+      return resp.body;
+    } catch (err) {
+      this.logger.error(err, 'failed to create outbound call from ' + this.callSid);
+    }
+  }
+
+  async _validateCreateCall(opts) {
```

Comment:
why do we need to validate here?  The endpoint we are calling `/createCall` will do this validation.  So it seems duplicated...can't we just capture the error response (typically a 400 Bad Request) and send it back over the websocket in some form?

Reply:
I removed the part of code for data validation, I just now keep the code for transforming data that feature server can understand and process the outbound call